### PR TITLE
Added table view

### PR DESCRIPTION
**Pull request by ElenSuvarianTer**
# What does this PR do?

It creates a table view for existing projects# Where should the reviewer start?

# How should this be manually tested?
In the projects view, click on the Table Icon

# Any background context you want to provide?

# What are the relevant tickets?
remix-project-lab #176:
"Have a table view for the projects, so we can see valuable information in a more structured and easy to read format"

# Screenshots

Captura de pantalla 2023-07-03 a las 10 39 53

# Questions

# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
         "@emotion/react": "^11.9.0",
         "@emotion/styled": "^11.8.1",
         "@google-cloud/bigquery": "^6.0.3",
-        "@mui/icons-material": "^5.8.4",
+        "@material-ui/core": "^4.12.4",
+        "@mui/icons-material": "^5.11.16",
```
Comment (martinrobled0) [inline package-lock.json:13]: why do we need that version?

Comment (ElenSuvarianTer) [reply package-lock.json:13]: we really don't need it.

Comment (ElenSuvarianTer) [reply package-lock.json:13]: Fixed!

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
     "@google-cloud/bigquery": "^6.0.3",
+    "@material-ui/core": "^4.12.4",
```
Comment (jackbravo) [inline package.json:33]: this shouldn't be necessary, we are already using mui v5.

**Review by jackbravo — CHANGES_REQUESTED** [review]

**Review by jackbravo — APPROVED** [review]

**Pull request merged**